pxc/jenkins/qa_pxc_cross_version_57_80_test-pipeline.groovy

```diff
@@ -0,0 +1,179 @@
+                    # Fetch the latest Upper_PXC binaries
+                    cd $ROOT_FS
+                    rm -rf $ROOT_FS/pxc_8.0_tar || true
+                    UPPER_PXC_TAR=upper-pxc-latest.tar.gz
+                    wget -qcO - https://downloads.percona.com/downloads/TESTING/pxc-${PXC_UPPER_VERSION_TAR}/Percona-XtraDB-Cluster_${PXC_UPPER_VERSION_TAR}_Linux.x86_64.glibc2.17.tar.gz > ${UPPER_PXC_TAR}
+                    tar -xzf ${UPPER_PXC_TAR}
+                    rm *.tar.gz
+                    mv Percona-XtraDB-* pxc_8.0_tar
+                ''' 
+  sh 'echo Starting cross verification script: $(date -u)'
+               sh '''
+                    set +e
+                    rm -rf percona-qa
+                    ROOT_FS=$PWD
+                    git clone ${PERCONA_QA_REPO} --branch ${BRANCH} --depth 1
+                    cd percona-qa/pxc-tests
+                    bash -x cross_version_pxc_57_80_test.sh $ROOT_FS/pxc_5.7_tar $ROOT_FS/pxc_8.0_tar
+                    error_code_1=$?
+                    set -e
+                    if [ "$error_code_1" = "0" ]; then
+                      exit 0
+                    else
+                      echo "#############################"
+                      echo "Printing PXC_57 error logs..."
+                      echo "#############################"
+                      cat /mnt/jenkins/workspace/qa_pxc_cross-pipeline/pxc_5.7_tar/pxc-node/node1.err
+                      echo "#############################"
+                      echo "Printing PXC_80 error logs..."
+                      echo "#############################"
+                      cat /mnt/jenkins/workspace/qa_pxc_cross-pipeline/pxc_8.0_tar/pxc-node/node2.err
+                      exit 1
+                    fi
+                '''
+  sh 'echo Starting cross verification upgrade script: $(date -u)'
+                sh '''
+                    set +e
+                    ROOT_FS=$PWD
+                    cd percona-qa/pxc-tests
+                    bash -x cross_version_pxc_57_80_upgrade_test.sh $ROOT_FS/pxc_5.7_tar $ROOT_FS/pxc_8.0_tar
+                    error_code_2=$?
+                    set -e
+                    if [ "$error_code_2" = "0" ]; then
+                      exit 0
+                    else
+                      echo "#############################"
+                      echo "Printing PXC_57 error logs..."
+                      echo "#############################"
+                      cat /mnt/jenkins/workspace/qa_pxc_cross-pipeline/pxc_5.7_tar/pxc_node_57_1/node_57_1.err
+                      cat /mnt/jenkins/workspace/qa_pxc_cross-pipeline/pxc_5.7_tar/pxc_node_57_2/node_57_2.err
+                      echo "#############################"
+                      echo "Printing PXC_80 error logs..."
+                      echo "#############################"
+                      cat /mnt/jenkins/workspace/qa_pxc_cross-pipeline/pxc_8.0_tar/pxc_node_80_1/node_80_1.err
```

Comment:
Fix these lines as the job name has changed. Please test this after updating. You can induce a script failure to verify if it is printing the error logs or not